app/Traits/CachesOrders.php

```
@@ -0,0 +1,107 @@
+            $roleSlug = 'administrator';
+        } elseif (method_exists($user, 'isSuperAdministrator') && $user->isSuperAdministrator()) {
+            $roleSlug = 'super_administrator';
+        }
+
+        if (!empty($filters)) {
+            ksort($filters);
+        }
+        $filtersPart = !empty($filters) ? ':filters:' . md5(json_encode($filters)) : '';
+
+        return "orders:index:v{$version}:role:{$roleSlug}:user:{$user->id}{$filtersPart}";
+    }
```

PHPMD (2.15.0)

The method indexKeyFor() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)